shell/common/shell.cc

```diff
@@ -610,6 +611,18 @@ void Shell::OnPlatformViewSetViewportMetrics(const ViewportMetrics& metrics) {
   FML_DCHECK(is_setup_);
   FML_DCHECK(task_runners_.GetPlatformTaskRunner()->RunsTasksOnCurrentThread());
 
+  // Only set this if the user has _not_ requested a specific value.
+  if (raster_cache_max_bytes_user_value_ == -1) {
+    // This is the formula Android uses.
+    int max_bytes = metrics.physical_width * metrics.physical_height * 12 * 4;
```

Comment:
How did we figure 12 times screen size was a good metric? Seems really large. But I am not sure what it should be. Maybe 3-4?

Reply:
The android dev who pointed this out to me suggested that it should be lower (I think his suggestion was 5).

I think it's probably fine to make it lower, but this is what Android has.